runtime/dali/src/lib.rs

```diff
@@ -867,20 +867,31 @@ impl crowdloan_rewards::Config for Runtime {
 
 parameter_types! {
 	pub const StakingRewardsPalletId : PalletId = PalletId(*b"stk_rwrd");
+	pub const MaxStakingDurationPresets : u32 = 10;
+}
+
+#[derive(PartialEq, Eq, Copy, Clone, Encode, Decode, MaxEncodedLen, TypeInfo, RuntimeDebug)]
+pub struct MaxRewardConfigsPerPool;
+impl Get<u32> for MaxRewardConfigsPerPool {
+	fn get() -> u32 {
+		10
+	}
 }
 
 impl pallet_staking_rewards::Config for Runtime {
 	type Event = Event;
-	type Share = Balance;
 	type Balance = Balance;
-	type PoolId = u16;
+	type RewardPoolId = u16;
 	type PositionId = u128;
-	type MayBeAssetId = CurrencyId;
+	type AssetId = CurrencyId;
```

Comment:
It is not asset, it is may be asset. please retain the difference or split these 2